README.md

```diff
@@ -127,6 +127,36 @@ checkerFramework {
 }
 ```
 
+### Multi-project builds
+
+By default, checkers are run on all subprojects of the project to which the plugin
+is applied. If you wish to disable this behavior, set the `applyToSubprojects`
+flag to `false` (note that disabling this behavior will require you to configure
+each subproject by hand in an appropriate `build.gradle` file):
+
+```groovy
+checkerFramework {
+  applyToSubprojects = false
+}
+```
+
+In most projects with subprojects, you will still want to avoid attempting to configure
```

Comment:
Should this paragraph go before the previous one?  It seems to be applicable most of the time, whereas the previous paragraph is an uncommon case.

Reply:
Good point. I have reordered the paragraphs.